homeassistant/components/trello/const.py

```diff
@@ -0,0 +1,15 @@
+"""Constants for the trello_int integration."""
+import logging
+from typing import Final
+
+DOMAIN: Final = "trello"
+
+LOGGER = logging.getLogger(__package__)
+
+CONF_API_TOKEN = "api_token"
+CONF_USER_ID = "user_id"
+CONF_USER_EMAIL = "user_email"
+
+CONF_OPTIONS_BOARDS = "boards"
+
+USER_INPUT_BOARD_IDS = "board_ids"
```

Comment:
```suggestion
CONF_BOARD_IDS = "board_ids"
```
I think this prefix works better. I see that you're using this only as temporary, to fetch the boards but I think you can just use 1 variable

Reply:
Yeah I'm kinda split on this. Although it's tempting to reduce duplication by reusing strings, the strings are really representing keys for different things. Though I'm not consistent with this myself since I use other `CONF_*` consts as keys on `user_input` [elsewhere](https://github.com/home-assistant/core/pull/93580/files#diff-a4883022abef83076d53d668dbc99a6a66fe90a152e0381b9ac04f818d05b729R57-R58).